couchbase/json-buckets.yml

```diff
@@ -0,0 +1,968 @@
+
+      - name: couch.spatial.disk.size
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_spatial_disk_size[0]
+        type: long_gauge
+        label: spatial disk size
+        unit: bytes
+        description: Amount of disk space occupied by spatial views
+
+      - name: couch.spatial.ops
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_spatial_ops[0]
+        type: long_gauge
+        label: spatial ops
+        description: Spatial operations
+
+      - name: couch.total.disk.size
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_total_disk_size[0]
+        type: long_gauge
+        label: total disk size
+        unit: bytes
+        description: Couch total disk size
+
+      - name: couch.views.data.size
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_views_data_size[0]
+        type: long_gauge
+        label: data size
+        unit: bytes
+        description: Size of object data for views
+
+      - name: couch_views_disk_size
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_views_disk_size[0]
+        type: long_gauge
+        label: disk size
+        unit: bytes
+        description: Amount of disk space occupied by views
+
+      - name: couch.views.fragmentation
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.couch_views_fragmentation[0]
+        type: long_gauge
+        label: view framentation
+        unit: "%"
+        description: View fragmentation
+
+      - name: cpu.idle.ms
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.cpu_idle_ms[0]
+        type: long_gauge
+        label: idle CPU
+        unit: milliseconds
+        description: CPU idle milliseconds
+
+      - name: cpu.utilization.rate
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.cpu_utilization_rate[0]
+        type: long_gauge
+        label: CPU utilization
+        unit: "%"
+        description: CPU utilization percentage
+
+      - name: curr.connections
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.curr_connections[0]
+        type: long_gauge
+        label: current connections
+        description: Current bucket connections
+
+      - name: curr.items.tot
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.curr_items_tot[0]
+        type: long_gauge
+        label: items
+        description: Total number of items
+
+      - name: curr.items
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.curr_items[0]
+        type: long_gauge
+        label: acive items
+        description: Number of active items in memory
+
+      - name: decr.hits
+        source: json:/pools/default/buckets/${bucketName}/stats $.op.samples.decr_hits[0]
+        type: long_gauge
```

Comment:
It is not counter, it gauge.

Reply:
Again something reported for last 1 sec :(? There are so many of these useless metrics... If we drop them all, it will look like we are missing things. If we keep them, they will be useless. Maybe we should... keep them all and document each of these that are problematic so people are aware of why they are bad, somebody might still find them useful in some way (maybe just to see a trend, if the numbers themselves are not reliable).